assets/xml/objects/object_boss07.xml

```diff
@@ -1,216 +1,270 @@
+        <Animation Name="gMajorasWrathKickAnim" Offset="0x26204" /> <!-- Original name is "last_kick" -->
+        <Animation Name="gMajorasWrathReleaseTopAnim" Offset="0x269EC" /> <!-- Original name is "last_koma" ("spinning top") -->
+        <Animation Name="gMajorasWrathGrabAnim" Offset="0x26EA0" /> <!-- Original name is "last_link" -->
+        <Animation Name="gMajorasWrathThrowAnim" Offset="0x27270" /> <!-- Original name is "last_link2" -->
+
+        <!-- Majora's Wrath Limb DisplayLists -->
+        <DList Name="gMajorasWrathPelvisDL" Offset="0x2C350" />
+        <DList Name="gMajorasWrathHeadDL" Offset="0x2C4D0" />
+        <DList Name="gMajorasWrathThirdEyeDL" Offset="0x2CB48" />
+        <DList Name="gMajorasWrathTorsoDL" Offset="0x2CD40" />
+        <DList Name="gMajorasWrathLeftHandDL" Offset="0x2D1D0" />
+        <DList Name="gMajorasWrathLeftForearmDL" Offset="0x2D3C8" />
+        <DList Name="gMajorasWrathLeftUpperArmDL" Offset="0x2D670" />
+        <DList Name="gMajorasWrathRightHandDL" Offset="0x2D940" />
+        <DList Name="gMajorasWrathRightForearmDL" Offset="0x2DB38" />
+        <DList Name="gMajorasWrathRightUpperArmDL" Offset="0x2DDE0" />
+        <DList Name="gMajorasWrathRightFootDL" Offset="0x2E0B8" />
+        <DList Name="gMajorasWrathRightShinDL" Offset="0x2E220" />
+        <DList Name="gMajorasWrathRightThighDL" Offset="0x2E4A8" />
+        <DList Name="gMajorasWrathLeftFootDL" Offset="0x2E6D0" />
+        <DList Name="gMajorasWrathLeftShinDL" Offset="0x2E838" />
+        <DList Name="gMajorasWrathLeftThighDL" Offset="0x2EAC0" />
+
+        <!-- Majora's Wrath Whip Texture and DisplayLists -->
+        <Texture Name="gMajorasWrathWhipTex" OutName="majoras_wrath_whip" Format="rgba16" Width="8" Height="16" Offset="0x2ECF0" />
+        <DList Name="gMajorasWrathWhipMaterialDL" Offset="0x2EE50" />
+        <DList Name="gMajorasMaskWhipShadowMaterialDL" Offset="0x2EEC8" />
+        <DList Name="gMajorasWrathWhipModelDL" Offset="0x2EEF8" />
+
+        <!-- Majora's Wrath Shadow DisplayLists -->
+        <DList Name="gMajorasWrathShadowMaterialDL" Offset="0x2EF68" />
+        <DList Name="gMajorasWrathShadowModelDL" Offset="0x2EF88" />
+
+        <!-- DisplayList for the rays of light that emanate from Wrath's body when defeated -->
+        <DList Name="gMajorasWrathDeathLightModelDL" Offset="0x2EFE8" />
+
+        <!-- Majora's Wrath Top DisplayList-->
+        <DList Name="gMajorasWrathTopDL" Offset="0x2F640" />
+
+        <!-- Majora Title Cards -->
+        <Texture Name="gMajorasMaskTitleCardTex" OutName="majoras_mask_title_card" Format="i8" Width="128" Height="40" Offset="0x2F840" />
+        <Texture Name="gMajorasIncarnationTitleCardTex" OutName="majoras_incarnation_title_card" Format="i8" Width="128" Height="40" Offset="0x30C40" />
+        <Texture Name="gMajorasWrathTitleCardTex" OutName="majoras_wrath_title_card" Format="i8" Width="128" Height="40" Offset="0x32040" />
```

Comment:
We should consider what to do language-wise here since we'll have others to deal with in due course.

Reply:
You mean what to do for other languages of the game? I suppose we could append "US" or even "NES" to the end of the names here to indicate they're for the US version. That lets us do something similar for Japanese and PAL. Should I do it in this PR for all the bosses?

Reply:
I'm a bit confused; they do the exact same thing in OoT:
```xml
<Texture Name="gBongoTitleCardTex" OutName="bongo_title_card" Format="i8" Width="128" Height="120" Offset="0x13D80"/>
```
Did you think they did something else?

Reply:
Oh right, it's one texture for some bizarre reason.

Reply:
Yeah, I wasn't sure what you wanted me to do here. I could put "US" here to say "hey, this is English only", but I definitely can't just put "English", since PAL's texture has English too.

Reply:
I think just leave it for now then.